### ISSUE TITLE
Cervical C6/C7 vertebral fusion

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10445]
**Preferred term label:**
Cervical C6/C7 vertebral fusion

**Synonyms**
-

**Definition (free text, please give PubMed ID)**
[based on HPO-term Cervical C3/C4 vertebral fusion HP:0030281]

Fusion of cervical vertebrae at C6 and C7, caused by a failure in the normal segmentation or division of the cervical vertebrae during the early weeks of fetal development.

**Parent term (use hpo.jax.org/app)**
Fused cervical vertebrae HP:0002949

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Apert syndrome (OMIM101200) PMID25206089
Bruck syndrome 2 (OMIM6009200) PMID31472299
Klippel-feil syndrome (OMIM613702) PMID: 17023841

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000807
- label: Cervical C6/C7 vertebrae fusion
- parents: [Fused cervical vertebrae (HP:0002949)](https://hpo.jax.org/app/browse/term/HP:0002949)
- pmid: PMID:25206089
- def: Fusion of cervical vertebrae at C6 and C7, caused by a failure in the normal segmentation or division of the cervical vertebrae during the early weeks of fetal development, leading to a short neck with a low hairline at the back of the head, and restricted mobility of the upper spine.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10445
- synonyms:
